edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -308,43 +283,58 @@ impl MakeModuleRuntime for DockerModuleRuntime {
     }
 }
 
-async fn create_network_if_missing(settings: &Settings, client: &DockerClient) -> Result<()> {
-    let (enable_ipv6, ipam) = get_ipv6_settings(settings.moby_runtime().network());
+async fn init_client(docker_url: &Url) -> Result<DockerApiClient> {
```

Comment:
I'm not sure how to initialize the hyper client now. @gordonwang0 do we have a helper function for this now or should I do it using the hyper builder function?